.gitlab-ci.yml

```diff
@@ -120,6 +121,22 @@ test-apm-injection:
   script:
     - ./test/dockertest.sh --image "registry.ddbuild.io/${IMAGE}" --script install_script_agent7.sh --injection "$INJECTION" --no-agent "$NO_AGENT" --apm-libraries "$APM_LIBRARIES"
 
+test-observability-pipelines-worker:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:docker"]
+  stage: test
+  dependencies: ["generate-scripts"]
+  parallel:
+    matrix:
+      - IMAGE: images/mirror/amazonlinux:2
+      - IMAGE: images/mirror/debian:10
+      - IMAGE: images/mirror/debian:11
+      - IMAGE: images/mirror/rockylinux:9.0
+      - IMAGE: images/mirror/ubuntu:20.04
+      - IMAGE: images/mirror/ubuntu:22.04
```

Comment:
Pulled from https://datadoghq.atlassian.net/wiki/spaces/OPE/pages/2970288333/Observability+Pipelines+Worker+Supported+Platform+and+Versions missing a centos/rhel 8